components/Metrics/AdminReusesPage.vue

```diff
@@ -0,0 +1,199 @@
+                    :href="getReuseAdminUrl(reuse)"
+                  >
+                    <TextClamp
+                      :text="reuse.title"
+                      :auto-resize="true"
+                      :max-lines="2"
+                    />
+                  </CdataLink>
+                </AdminContentWithTooltip>
+              </td>
+              <td class="font-mono text-right">
+                {{ summarize(reuse.metrics.discussions) }}
+              </td>
+              <td class="font-mono text-right">
+                {{ summarize(reuse.metrics.views) }}
+              </td>
+              <td class="font-mono text-right">
+                {{ summarize(reuse.metrics.followers) }}
+              </td>
+              <td>
+                <BrandedButton
+                  size="xs"
+                  color="secondary-softer"
+                  :href="getCsvUrl(reuse)"
+                  :icon="RiDownloadLine"
+                  icon-only
+                  external
+                  keep-margins-even-without-borders
+                >
+                  {{ $t("Télécharger les statistiques de la réutilisation") }}
+                </BrandedButton>
+              </td>
+            </tr>
+          </tbody>
+        </AdminTable>
+        <Pagination
+          :page="page"
+          :page-size="pageSize"
+          :total-results="pageData.total"
+          @change="(changedPage: number) => page = changedPage"
+        />
+      </div>
+    </LoadingBlock>
+    <div
+      v-if="status != 'pending' && pageData && !pageData.total"
+      class="flex flex-col items-center"
+    >
+      <nuxt-img
+        src="/illustrations/reuse.svg"
+        class="h-20"
+      />
+      <p class="fr-text--bold fr-my-3v">
+        {{ $t(`Vous n'avez pas encore publié de réutilisation.`) }}
+      </p>
+      <AdminPublishButton type="reuse" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { RiDownloadLine, RiStarSLine } from '@remixicon/vue'
+import { BrandedButton, Pagination, summarize, Tooltip, type Organization, type Reuse, type User } from '@datagouv/components-next'
+import AdminTable from '~/components/AdminTable/Table/AdminTable.vue'
+import AdminTableTh from '~/components/AdminTable/Table/AdminTableTh.vue'
+import type { PaginatedArray, ReuseSortedBy, SortDirection } from '~/types/types'
+
+const props = defineProps<{
+  organization?: Organization
+  user?: User
+}>()
+
+const config = useRuntimeConfig()
+
+const downloadStatsUrl = computed(() => props.organization ? `/api/1/organizations/${props.organization.id}/reuses.csv` : null)
+
+const page = ref(1)
+const pageSize = ref(20)
+const sortedBy = ref<ReuseSortedBy>('created')
+const direction = ref<SortDirection>('desc')
+const sortDirection = computed(() => `${direction.value === 'asc' ? '' : '-'}${sortedBy.value}`)
+
+const params = computed(() => {
+  return {
+    organization: props.organization?.id,
+    user: props.user?.id,
```

Comment:
I don't understand why but I see all the reuses of the application… Same with dataservices. Datasets are working… Maybe a problem with the new api fields in udata?

Reply:
I think the search are missing the `user`, I'll make a PR on the udata  side